packages/core/src/components/button/test/basic/button.e2e.ts

```diff
@@ -3,97 +3,40 @@ import { expect } from '@playwright/test';
 
 const componentTestPath = 'src/components/button/test/basic/index.html';
 
-test.describe('tds-button', () => {
+test.describe('tds-button-basic', () => {
   test('renders basic button correctly', async ({ page }) => {
     await page.goto(componentTestPath);
-    const button = page.locator('tds-button');
-    await expect(button).toHaveScreenshot({ maxDiffPixels: 0 });
+    const button = page.getByTestId('tds-button-testid');
```

Comment:
Question: should we use `getByRole` here also? As we can. Or does using `getByTestId` here provide anything else?

Reply:
Erm yeah I can see few. the test id selects `tds-button` while `.getByRole('button')` would get the <button> element. So its really what we would like to test here. Check if tds-button is in the dom or if it has a <button> inside of it...

Reply:
Updated comment didnt see the `<button>` dissapeared at first. But no problem, I guess we have to decide on a pattern here on what we would like to test on the first test with the screenshot.

Reply:
Yeah! What do you think @theJohnnyMe and @alexandereneroth?